zebra-rpc/src/methods.rs

```diff
@@ -35,10 +51,15 @@ pub trait Rpc {
 pub struct RpcImpl;
```

Comment:
Suggestion (1/3): storing the string in the RPC method object seems a bit simpler than a lazy_static and mutex.

```suggestion
pub struct RpcImpl {
    /// Zebra's application version.
    pub app_version: String,
}
```